data/host.json

```diff
@@ -2,7 +2,7 @@
   "rhelHost": {
     "metadata": {
       "resource_type": "rhel-host",
-      "workspace": ""
+      "workspace": "workspace1"
```

Comment:
Could you also update the [k8s-cluster](https://github.com/project-kessel/inventory-api/blob/0be975b3336ba576b96ef8b6ababe27e7d089759/data/k8s-cluster.json#L5) example?

Reply:
I intentionally did not update it because that's for on-prem right now, which will use allow-all, and it may cause more confusion if we do?

Reply:
oh, that makes sense